iconservice/prep/data/prep_container.py

```diff
@@ -151,18 +154,18 @@ def replace(self, new_prep: 'PRep') -> Optional['PRep']:
 
         return old_prep
 
-    def contains(self, address: 'Address', active_prep_only: bool = True) -> bool:
+    def contains(self, address: 'Address', active_suspended_prep_only: bool = True) -> bool:
```

Comment:
To retain consistency, integrate 'active_suspended_prep_only' default value to False.

Reply:
spec is changed, that container defined ACTIVE = ACTIVE | SUSPENDED.